ui/src/main/webapp/src/app/analysis-context/analysis-context-form.component.html

```diff
@@ -89,8 +89,10 @@ <h2 *ngIf="!project" i18n>Loading...</h2>
                     <ng-container i18n>Identifying packages...</ng-container>
                 </p>
             </div>
-            <wu-js-tree-wrapper [treeNodes]="packageTree" [(selectedNodes)]="analysisContext.includePackages">
-            </wu-js-tree-wrapper>
+            <div class="fixed-height">
+                <wu-js-tree-wrapper [treeNodes]="packageTree" [(selectedNodes)]="analysisContext.includePackages">
+                </wu-js-tree-wrapper>
+            </div>
```

Comment:
@klinki actually it is valid for all usage, except you would like to specify the height directly, or not?

@mrizzi I tried to add it in jstree component and it worked for me